requirements.txt

```diff
@@ -0,0 +1,26 @@
+--extra-index-url https://rospypi.github.io/simple/
```

Comment:
is this still needed?

Reply:
Yes this is the link to all the packages in the index: https://rospypi.github.io/simple/

Two examples of packages not in the pypi index that are in simple that we end up using:
- rospy-builder
- std-msgs